src/main/java/org/openstreetmap/atlas/proto/adapters/ProtoIntegerStringDictionaryAdapter.java

```diff
@@ -0,0 +1,80 @@
+package org.openstreetmap.atlas.proto.adapters;
+
+import org.openstreetmap.atlas.exception.CoreException;
+import org.openstreetmap.atlas.proto.ProtoIntegerStringDictionary;
+import org.openstreetmap.atlas.proto.ProtoSerializable;
+import org.openstreetmap.atlas.utilities.compression.IntegerDictionary;
+
+import com.google.protobuf.InvalidProtocolBufferException;
+
+/**
+ * Implements the {@link ProtoAdapter} interface to connect {@link IntegerDictionary} and
+ * {@link ProtoIntegerStringDictionary}. This adapter will fail when attempting to serialize an
+ * {@link IntegerDictionary} that is not type parameterized using {@link String}.
+ *
+ * @author lcram
+ */
+public class ProtoIntegerStringDictionaryAdapter implements ProtoAdapter
+{
+    // NOTE this method will ALWAYS give back an IntegerDictionary<String>, no matter what type
```

Comment:
Nit: This may be important enough to add to the javadoc so anyone using this class can see it.